dotnet/src/Functions/Functions.OpenAPI/Builders/Serialization/FormStyleParameterSerializer.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json.Nodes;
+using System.Web;
+using Microsoft.SemanticKernel.Diagnostics;
+using Microsoft.SemanticKernel.Functions.OpenAPI.Model;
+
+namespace Microsoft.SemanticKernel.Functions.OpenAPI.Builders.Serialization;
+
+/// <summary>
+/// Serializes REST API operation parameter of the 'Form' style.
+/// </summary>
+internal static class FormStyleParameterSerializer
+{
+    /// <summary>
+    /// Serializes a REST API operation `Form` style parameter.
+    /// </summary>
+    /// <param name="parameter">The REST API operation parameter to serialize.</param>
+    /// <param name="argument">The parameter argument.</param>
+    /// <returns>The serialized parameter.</returns>
+    public static string Serialize(RestApiOperationParameter parameter, string argument)
+    {
+        const string ArrayType = "array";
+
+        if (parameter is null)
+        {
+            throw new ArgumentNullException(nameof(parameter));
+        }
+
+        if (parameter.Style != RestApiOperationParameterStyle.Form)
+        {
+            throw new SKException($"Unexpected Rest Api operation parameter style - `{parameter.Style}`");
+        }
+
+        // Handling parameters of array type.
+        if (parameter.Type == ArrayType)
```

Comment:
Any reason not to use an enum here?

Reply:
No particular reason, it should have been added as many other things, but it was never done. I add it to my TODO list to address it after I'm done with the "array" support for query string parameters.